modules/custom/az_person/config/install/core.entity_view_display.node.az_person.az_row.yml

```
@@ -141,6 +141,21 @@ third_party_settings:
         attributes: ''
         effect: none
         speed: fast
+    group_link:
+      children:
+        - field_az_media_image
+      label: 'Image Link'
+      parent_name: group_col_image
+      region: content
+      weight: 23
+      format_type: link
+      format_settings:
+        classes: hide-contextual-links
```

Add this back because: https://github.com/az-digital/az_quickstart/blob/2.5.x/themes/custom/az_barrio/templates/content/node--az-person--az_row.html.twig#L87
```suggestion
        classes: 'hide-contextual-links az-person-row-img'
```